src/main/kotlin/it/pagopa/wallet/eventdispatcher/queues/WalletCdcQueueConsumer.kt

```diff
@@ -36,16 +39,18 @@ class WalletCdcQueueConsumer(
     private val logger = LoggerFactory.getLogger(WalletCdcQueueConsumer::class.java)
     private val consumerSpanName = WalletCdcQueueConsumer::class.java.simpleName
 
+    fun parseEvent(payload: ByteArray): Mono<CdcQueueEvent<LoggingEvent>>? {
```

Comment:
```suggestion
    fun parseEvent(payload: ByteArray): Mono<CdcQueueEvent<LoggingEvent>> {
```

remove nullable return value here